CupOfJoin.py

```diff
@@ -0,0 +1,9 @@
+def join(*lists, sep='-'):
```

Comment:
Add type hints and type definitions  - read here about how its done - https://docs.python.org/3/library/typing.html

Reply:
![Capture](https://user-images.githubusercontent.com/13467749/160146959-11960c01-f4d4-484e-8c7c-f2cb80f02b1d.JPG)
it should look like this